specs/waku/v2/waku-filter.md

```diff
@@ -38,28 +44,15 @@ It is worth noting that a light node could get by with only using the `store`
 protocol to query for a recent time window, provided it is acceptable to do
 frequent polling.
 
-#### Protobuf
-
-TODO Consider adding a FilterResponse acting as a form of ACK
-
-TODO Consider adding request id
-
-TODO Clarify if Messages or a list of WakuMessage are pushed
-
-TODO Specify unsubscribe mechanism and semantics
-
-TODO Investigate if we need a way to communicate (handshake?) that we are a a client - server (full node - light node) or not.
-NOTE I would imagine this is implied from the contentFilter, especially as two nodes can play multiple roles.
-
+## Protobuf
 
 ```protobuf
 message FilterRequest {
-  // space for optional request id
-  repeated ContentFilter contentFilters = 2;
-  optional string topic = 3;
+  string topic = 1;
```

Comment:
This is the WakuRelay pubsub topic right?

I understand why you add it, but I'm still wondering whether we need that additional flexibility.

Reply:
Because the topics might be different. We discussed this in a call I believe, essentially a filter is made up of pubsub topic and content filter. Omitting this would lead to a lot of unspecified behavior, especially as we move to multiple pubsub topics I believe.